x-pack/plugins/reporting/server/export_types/printable_pdf/execute_job/index.ts

```diff
@@ -42,9 +42,7 @@ export const runTaskFnFactory: QueuedPdfExecutorFactory = function executeJobFac
       mergeMap(() => decryptJobHeaders({ encryptionKey, job, logger })),
       map((decryptedHeaders) => omitBlacklistedHeaders({ job, decryptedHeaders })),
       map((filteredHeaders) => getConditionalHeaders({ config, job, filteredHeaders })),
-      mergeMap((conditionalHeaders) =>
-        getCustomLogo({ reporting, config, job, conditionalHeaders })
-      ),
+      mergeMap((conditionalHeaders) => getCustomLogo(reporting, conditionalHeaders, job.spaceId)),
```

Comment:
I refactored `getCustomLogo` to receive more specific parameters. The other common helper functions here could probably use the same thing